src/components/profile/components/LeaderboardRow.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const LeaderboardRow = ({ data, isSelected, onClick }) => {
+    return (
+        <li
+            className={`mr-2 flex justify-between items-center p-4 rounded-md cursor-pointer ${isSelected ? 'bg-cyan-950' : 'bg-gray-100'}`}
+            onClick={onClick}
+        >
+            <div className="flex items-center space-x-2">
+                <span
+                    className={`font-normal text-xs w-3 ${isSelected ? 'text-white' : 'text-gray-700'}`}
+                >
+                    {data.id}
+                </span>
+                <hr className="w-6 rotate-90 border-white p-0" />
+                <img
+                    src="https://picsum.photos/40"

```

Comment:
@devowl03 You need to take this profilePic, or now when we are using metamask now then generated image from data. This is unique for each user in leaderboard, can't keep same for everyone.

@vaibhavgeek please confirm the requirements.